tutorials/napps/publishing_your_napp.rst

```diff
@@ -179,49 +172,52 @@ Uploading the NApp
 ******************
 
 Your NApp is now ready to be uploaded. 
-Before that, if you do not want to send some files to the server, add them to .gitignore.
-To upload your Napp, use the following command:
+Before that, you need to create a repository with NApp name on github.
+After creating repository follow the below commands to initialize and push your NApp to github.
 
 .. code-block:: console
 
-  $ cd ~/tutorials/<username>/my_first_napp
-  $ kytos napps upload
-  Enter the username: <username>
-  Enter the password for <username>: <password>
-  SUCCESS: NApp <username>/my_first_napp uploaded.
+  $ cd ~/tutorials/<username>/<name>
+  $ git init
+  $ git remote add origin https://github.com/<username>/<name>.git
+  $ git add .
+  $ git push -u origin master
 
-You must fill **username** and **password** with the values you used to
-register your user on the |napps_server|_.
+You will be asked to input your username and password of github, here password indicates generated token from github, View this |github_token|_ to generate your token.
 
-Your NApp is now uploaded. You can see it on the web
-(`<https://napps.kytos.io/<username>/my_first_napp`) or search for it through
-the command line with the following command:
 
-.. code-block:: console
+  Enter username: <username>
 
-  $ kytos napps search <username>/my_first_napp
+  Enter password: <token>
 
-  Status |       NApp ID            |                     Description
-  =======+==========================+====================================================
-   [--]  | <username>/my_first_napp | This is my first NApp. I've built it as an examp...
+After finishing the above steps your napp will be pushed to github and your will be able to view it in your repository.
+Follow the instructions on this |github_topic|_ to include topic name 'kytos-napp' to your repository.
 
 
-  Status: (i)nstalled, (e)nabled
+Your NApp is now uploaded. You can see your NApp on the web: https://github.com/topics/kytos-napp
+
 
 ************************************
 Search for and Install a remote NApp
 ************************************
 
-Now that you have published your NApp, you can install it by using the command
-``kytos napps install <username>/my_first_napp``.
+Now that you have published your NApp, you can find your NApp |kytos_page|_, you can also look for other NApps published.
```

Comment:
```suggestion
Now that you have published your NApp, you can find your NApp on its GitHub page. Also, you can search and discover other NApps that have been published on GitHub |kytos_page|_
```

I think this new updated phrase emphasizes more how users can still search for it, let me know what you think or if you have other suggestions

Reply:
May be GitHub has paginated functionality, I don't see a search box in the Github topics window, However we are indicating the end users to find there Napp at Github topic page, do you suggest any other phrase ?

Reply:
Yeah, I think it'll be helpful to highlight both things here, users can find their NApps directly on its page aka http://github.com/<username>/my_first_napp, and then the topic itself even though they can be found there, this is meant for discoverability, that's the goal we're trying to achieve with the GitHub `topic` here, just so users could find other NApps by browsing the NApps, that's wy I suggested that phrase in my previous comment, feel free to reword differently, as long as we can convey both two points here that should be ok.

Reply:
Ok.